server/paths/manage.ts

```diff
@@ -132,8 +132,8 @@ const v2Manage = {
     create: outOfServiceBedsPath,
     premisesIndex: v2SinglePremisesPath.path('out-of-service-beds').path(':temporality'),
     index: outOfServiceBedsIndexPath.path(':temporality'),
-    update: singlePremisesPath.path('out-of-service-beds').path(':id'),
     show: outOfServiceBedPath.path(':tab'),
+    update: outOfServiceBedPath.path('/update'),
     cancel: singlePremisesPath.path('out-of-service-beds').path(':id').path('cancellations'),
   },
```

Comment:
Just wondering whether more smaller things might be easier to manage? See my later question about whether an entirely new "resource" might be clearer? e.g.  something like:

```ts
const outOfServiceBedsUpdatesPath = outOfServiceBedsIndexPath.path(':outOfServiceBedId')
// Path<"/manage/out-of-service-beds/:outOfServiceBedId">

outOfServiceBedUpdates: {
    new: outOfServiceBedsUpdatesPath.path('new'),
    create: outOfServiceBedsUpdatesPath,
  },
```